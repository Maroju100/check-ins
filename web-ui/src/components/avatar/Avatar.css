.avatar {
    width: 40px;
    height: 40px;
    background-color: #72c7d5;
    border-radius: 50%;
}