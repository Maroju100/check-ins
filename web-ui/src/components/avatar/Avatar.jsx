import React from 'react';
import './Avatar.css';

const Avatar = () => (
    <div className="avatar"></div>
);

export default Avatar;