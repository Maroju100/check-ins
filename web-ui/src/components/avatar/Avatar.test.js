import React from "react";
import Avatar from "./Avatar";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Avatar />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders image_url", () => {
  const tree = renderer
    .create(<Avatar image_url={"http://someurl.com/das.png"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
