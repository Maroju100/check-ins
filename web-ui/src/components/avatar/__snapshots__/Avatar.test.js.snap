// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiAvatar-root MuiAvatar-circle"
  style={
    Object {
      "backgroundColor": "#72c7d5",
      "cursor": "pointer",
      "position": "absolute",
      "right": "5px",
      "top": "10px",
    }
  }
>
  <img
    className="MuiAvatar-img"
    src="/default_profile.jpg"
  />
</div>
`;
